objectRecognitionSource/README.md

```diff
@@ -279,6 +279,95 @@ Options available for all Queries (not prepended by anything) are:
+        If it is defined here as a query parameter, it will override the value set in the source configuration.
+    
+*   **Delete locally saved images:**
+    *   The user can specify one, or multiple locally saved images to be deleted. The images will be those 
+    that are saved in the output directory, which is defined in the source configuration.
+    *   Parameters:
+        *   "operation": Required. Must be set to "delete".
+        *   *Only one of the following two values *MUST* be specified, or a query error will be returned. If both values are 
+        specified, the imageName value will be used. (For optimal use, only set one of these values.)*
+            *   "imageName": A string value representing the name of the file to be deleted. If set to "all", then all images 
+            in the output directory will be deleted.
+            *   "imageDate": A list containing two strings, a start date and an end date. All locally saved images falling 
+            between the start and end date, (inclusive), will be deleted. Dates must be formatted in the following 
+            manner: "yyyy-MM-dd--HH-mm-ss".
+
+*   **Process a single frame from the camera defined in the source configuration:**
+    *   Parameters:
+        *   "operation": Required. Must be set to "processNextFrame".
+        *   "NNsaveImage": Optional. This value can be set exactly like the "saveImage" value in the source configuration.
+        *   "NNoutputDir": Optional. This value can be set exactly like the "outputDir" value in the source configuration.
+        *   "NNfileName": Optional. A string representing the unique name used to save the file. If not specified, the file 
+        will be named using the standard <yyyy-MM-dd--HH-mm-ss.jpg> value.
+    
+**EXAMPLE QUERIES**:
+
+*   Upload Query using imageName:
+```
+SELECT * FROM SOURCE Camera1 AS results WITH
+    	operation:"upload",
+    	imageName:"2019-02-08--10-33-36.jpg",
+    	savedResolution: {longEdge=600}
+```
+
+*   Upload Query using imageDate:
+```
+SELECT * FROM SOURCE Camera1 AS results WITH
+    	operation:"upload",
+    	imageDate:["2019-02-08--10-33-36", "2019-02-08--10-34-06"]
+```
+
+*   Delete Query using imageName:
+```
+SELECT * FROM SOURCE Camera1 AS results WITH
+    	operation:"delete",
+    	imageName:"all"
+```
+
+*   Delete Query using imageDate:
+```
+SELECT * FROM SOURCE Camera1 AS results WITH
+    	operation:"delete",
+    	imageDate:["2019-02-08--10-33-36", "2019-02-08--10-34-06"]
+```
+
+*   Process Next Frame Query:
+```
+SELECT * FROM SOURCE Camera1 AS results WITH
+        operation:"processNextFrame",
+    	NNsaveImage:"local",
+    	NNoutputDir:"testDir",
+    	NNfileName:"testFile"
+```
+
+*   Process Next Frame Query without specifying operation *(not recommended)*:
+```
+SELECT * FROM SOURCE Camera1 AS results WITH
+    	NNsaveImage:"local",
+    	NNoutputDir:"testDir",
+    	NNfileName:"testFile"
+```
```

Comment:
Need at least one example with open end. Like `imageDate: ["-", "2019-02-08--10-33-36"]`

Reply:
I think we should add one that shows how to delete or upload everything.

*   Delete Query using imageDate to delete all images:
	```
	SELECT * FROM SOURCE Camera1 AS results WITH
	    	operation:"delete",
	    	imageDate:["-", "-"]
	```